sdk/core/azure-core/src/http/curl/curl.cpp

```diff
@@ -954,17 +954,28 @@ CURLcode CurlSession::ReadStatusLineAndHeadersFromRawResponse(
     // /3 containing a "Connection" header should be considered malformed. (HTTP/2:
     // https://httpwg.org/specs/rfc9113.html#ConnectionSpecific
     //  HTTP/3: https://httpwg.org/specs/rfc9114.html#rfc.section.4.2)
-    if (m_response->GetMajorVersion() == 1 && m_response->GetMinorVersion() >= 1)
+    // We assume that the server is not sending malformed responses,
+    // so we are not considering HTTP/2 or HTTP/3 here.
+    if (m_response->GetMajorVersion() == 1 && m_response->GetMinorVersion() == 1)
```

Comment:
This should be greater than as in the original change. 1.2 can not break the functionality of 1.1,

Reply:
Is there an HTTP version 1.2? I don't see anything about this, we have 1.1 and then 2.0:
https://en.wikipedia.org/wiki/HTTP#Summary_of_HTTP_milestone_versions